code/_helpers/_qdel.dm

```diff
@@ -0,0 +1,20 @@
+#define QDELING(X) (X.gc_destroyed)
+#define QDELETED(X) (!X || QDELING(X))
+#define QDESTROYING(X) (!X || X.gc_destroyed == GC_CURRENTLY_BEING_QDELETED)
+
+#define QDEL_IN(item, time) addtimer(CALLBACK(GLOBAL_PROC, .proc/qdel, item), time, TIMER_STOPPABLE)
+#define QDEL_IN_CLIENT_TIME(item, time) addtimer(CALLBACK(GLOBAL_PROC, .proc/qdel, item), time, TIMER_STOPPABLE | TIMER_CLIENT_TIME)
+#define QDEL_NULL(item) qdel(item); item = null
```

Comment:
#define QDEL_NULL(x) if(x) { qdel(x) ; x = null }

Reply:
> #define QDEL_NULL(x) if(x) { qdel(x) ; x = null }

This define deleted from defines/qdel.dm

Reply:
I prefer the old definition. why change it?

Reply:
> I prefer the old definition. why change it?

Because I extended the Qdel functions and removed the extra characters? For you, the value of the returned action will not change.